### ISSUE TITLE
Regarding content and style layers

### ISSUE THREAD

**Issue by kimdanny** [AI-School-Team4/Style-Transfer#3]
According to the paper from Gatys, et.al.(2015) and tensorflow ST tutorial, content_layer for VGG19 should be used from 'block5_conv2' and higher conv layers (conv1's) for style_layers. However, none of the empirical results we've done so far have proved this theory although we  printed out all the layers output images. Rather, its performance peaked when ["block1_conv2", "block2_conv2", "block3_conv3", "block4_conv3", "block5_conv3"] are used for style_layers.

There must be some reasons that we could not even realise until now....

Probably sth related to that freakin weighting issue....as it seems that we use the same loss functions
